File: testprojects/python_main/pipelines/PERF_UNITEST_GENERATE/code/perf_unitest_generate/graph/Subgraph_1/Reformat_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from perf_unitest_generate.config.ConfigStore import *
from perf_unitest_generate.udfs.UDFs import *

def Reformat_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: testprojects/scala_main/pipelines/CloneUTGenerate_PerfPipeline/code/src/main/scala/io/prophecy/pipelines/perf_unitest_generate/graph/Subgraph_1/Reformat_2.scala
package io.prophecy.pipelines.perf_unitest_generate.graph.Subgraph_1

import io.prophecy.libs._
import io.prophecy.pipelines.perf_unitest_generate.config.ConfigStore._
import io.prophecy.pipelines.perf_unitest_generate.config.Context
import io.prophecy.pipelines.perf_unitest_generate.udfs.UDFs._
import io.prophecy.pipelines.perf_unitest_generate.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_2 { def apply(context: Context, in: DataFrame): DataFrame = in }